### PR TITLE
kie-issues#126: Drill-down menus not working on KIE Sandbox after PatternFly upgrade (kie-issues#108)

### LINKED ISSUE #126: Drill-down menus not working on KIE Sandbox after PatternFly upgrade (kie-issues#108)
Clicking on drill-down menus to move forward is not working (nothing happens, nothing is logged).
Moving backward seems fine.

Image

Introduced on https://github.com/kiegroup/kie-issues/issues/108

### PR DESCRIPTION
**Pull request by thiagoelg**
- Fixes [kie-issues#126](https://github.com/kiegroup/kie-issues/issues/126)
- Remove sword breaking from menu item descriptions

### CODE REVIEW

**Comment by thiagoelg** [comment]
@caponetto You might want to check this :)

I've also fixed some menu item descriptions cutting words in half with a `!important` css prop, hopefully it won't affect anything else.

| Before | After |
|---|---|
| image|image|

**Review by pefernan — APPROVED** [review]
missed this! @thiagoelg good catch!

File: packages/serverless-logic-web-tools/src/editor/FileSwitcher.tsx [hunk 1]
```diff
@@ -181,21 +181,25 @@ export function FileSwitcher(props: { workspace: ActiveWorkspace; workspaceFile:
...
     setMenuDrilledIn((prev) => prev.slice(0, prev.length - 1));
     setDrilldownPath((prev) => prev.slice(0, prev.length - 1));
     setActiveMenu(toMenuId);
   }, []);
 
   const setHeight = useCallback((menuId: string, height: number) => {
-    // do not try to simplify this ternary's condition as some heights are 0, resulting in an infinite loop.
-    setMenuHeights((prev) => (prev[menuId] === height ? prev : { ...prev, [menuId]: height }));
+    setMenuHeights((prev) => {
+      if (prev[menuId] === undefined || (menuId !== "addFileRootMenu" && prev[menuId] !== height)) {
```
Comment (jstastny-cz) [inline packages/serverless-logic-web-tools/src/editor/FileSwitcher.tsx]: shouldn't the menuId value be replaced with const `ROOT_MENU_ID` ?

Comment (thiagoelg) [reply packages/serverless-logic-web-tools/src/editor/FileSwitcher.tsx]: Yep, nice catch! Thanks!

**Review by tiagobento — APPROVED** [review]

**Pull request merged**